### PR TITLE
PREAPPS-4814 : Briefcase | Upload files to briefcase from email

### PR DESCRIPTION
**Pull request by auminpatel**

### CODE REVIEW

File: src/normalize/entities.ts [hunk 1]
```diff
@@ -470,12 +470,18 @@ export const Document = new Entity({
 	ct: 'contentType'
 });
 
+export const MessagePartInputForDocuments = new Entity({
```
Comment (silentsakky) [inline src/normalize/entities.ts:473]: can we use MimePartInput ? it looks same

Comment (silentsakky) [reply src/normalize/entities.ts:473]: or ExistingAttachmentInput

Comment (auminpatel) [reply src/normalize/entities.ts:473]: actually we cant use , as MimePartInput or ExistingAttachmentInput uses messageId as mid for mapping , whereas in SaveDocument it uses id for messageId

**Comment by auminpatel** [comment]
closing this as created new PR

**Pull request closed without merging**